src/ui/pages/CredCardDetails/CredCardDetails.test.tsx

```diff
@@ -107,193 +107,168 @@ const initialStateNoPasswordArchived = {
 
 describe("Cards Details page - current not archived credential", () => {
   let storeMocked: Store<unknown, AnyAction>;
-  // beforeAll(() => {
-  // jest
-  //   .spyOn(AriesAgent.agent.credentials, "getCredentialDetailsById")
-  //   .mockResolvedValue(credsFixW3c[0]);
-  // });
-  // beforeEach(() => {
-  //   const mockStore = configureStore();
-  //   const dispatchMock = jest.fn();
-  //   storeMocked = {
-  //     ...mockStore(initialStateNoPasswordCurrent),
-  //     dispatch: dispatchMock,
-  //   };
-  // });
-
-  // test("It renders Card Details", async () => {
-  //   const { getByTestId, getAllByTestId } = render(
-  //     <Provider store={storeMocked}>
-  //       <MemoryRouter initialEntries={[path]}>
-  //         <Route
-  //           path={path}
-  //           component={CredCardDetails}
-  //         />
-  //       </MemoryRouter>
-  //     </Provider>
-  //   );
-  //   await waitFor(() => {
-  //     expect(getByTestId("creds-options-modal").getAttribute("is-open")).toBe(
-  //       "false"
-  //     );
-  //     expect(getByTestId("view-creds-modal").getAttribute("is-open")).toBe(
-  //       "false"
-  //     );
-  //     expect(getAllByTestId("verify-password")[0].getAttribute("is-open")).toBe(
-  //       "false"
-  //     );
-  //   });
-  // });
+  beforeAll(() => {
+    jest
+      .spyOn(Agent.agent.credentials, "getCredentialDetailsById")
+      .mockResolvedValue(credsFixAcdc[0]);
+  });
+  beforeEach(() => {
+    const mockStore = configureStore();
+    const dispatchMock = jest.fn();
+    storeMocked = {
+      ...mockStore(initialStateNoPasswordCurrent),
+      dispatch: dispatchMock,
+    };
+  });
 
-  // test("It copies id to clipboard", async () => {
```

Comment:
Why was this test removed?

Reply:
This unit test is copying the w3c.proofValue, but acdc didn't have that, So I removed it